composer.json

```diff
@@ -17,6 +17,8 @@
     ],
     "require": {
         "php": "^7.2",
+        "ext-simplexml":" *",
```

Comment:
This dependencies are optional (if someone uses it only for json), thue should be removed